Scheduler/observing_blocks_maintel/AOS/BLOCK-T160.json

```diff
@@ -0,0 +1,65 @@
+{
+    "name": "BLOCK-T160",
+    "program": "BLOCK-T160",
+    "reason": "tickets/SITCOM-1596",
+    "note": "wavefront_estimation_chromaticity",
+    "constraints": [],
+    "configuration_schema": "$schema: http://json-schema.org/draft-07/schema#\ntitle: BLOCK-T160 Configuration\ndescription: Configuration for the BLOCK-T160 Chromaticity Test WET-008.\ntype: object\nproperties:\n  ra:\n    description: Right Ascension.\n    type: number\n    default: 0\n  dec:\n    description: Declination.\n    type: number\n    default: 0\n  exposure_time:\n    description: Exposure Time.\n    type: number\n    default: 30.0\n",
+    "_schema_comment": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "BLOCK-T160 Configuration",
+    "description": "Configuration for the BLOCK-T160 Chromaticity Test WET-008.",
+    "type": "object",
+    "properties": {
+      "ra": {
+        "description": "Right Ascension.",
+        "type": "number",
+        "default": 0
+      },
+      "dec": {
+        "description": "Declination.",
+        "type": "number",
+        "default": 0
+      },
+      "exposure_time": {
+        "description": "Exposure Time.",
+        "type": "number",
+        "default": "30.0"
+      }
+    }
+  },
+    "scripts": [
+        {
+            "name": "maintel/enable_comcam.py",
+            "standard": true,
+            "parameters": {}
+        },
+        {
+            "name": "maintel/track_target.py",
+            "standard": true,
+            "parameters": {
+                "slew_icrs": {
+                    "ra": "$ra", 
+                    "dec": "$dec"
+                },
+                "rot_type": "PhysicalSky"
+            }
+        },
+        {
+            "name": "maintel/take_triplet_comcam.py",
+            "standard": true,
+            "parameters": {
+                "exposure_time": "$exposure_time",
+                "n_triplets": 2,
+                "program": "$program",
+                "reason": "tickets/SITCOM-1596",
```

Comment:
reason should be SITCOM-1596 by itself, but also you should use the ticket for analysis is actually SITCOM-1150